### DISCUSSION TITLE
Buscar polígono por [Estados|Cidades|Bairros]

### DISCUSSION THREAD

**Discussion by 3runoDesign** [BrasilAPI/BrasilAPI#244] [Ideas]
E ai! Parabéns pela iniciativa, excelente projeto!

Esses dias me deparei com a seguinte questão: "Se teríamos como pegar os polígono dos bairros de alguma cidade?"

Único serviço que encontrei de "forma aberta" com os bairros "desenhados" foi do [*Wikimapia*](https://wikimapia.org/#lang=pt&lat=-7.200619&lon=-48.206892&z=14&m=w&tag=4621)

Até consegui esses valores... de uma forma simples digamos assim.

Acreditam ser relevante para esse projeto?
Valeu!

**Comment by julioflima** [comment]
Muito interessante. 

Viajo muito e sempre me venho a questão sobre de quem que é esse terreno. Existe um projeto de lei que impede a União de possuir terrenos que dá a possibilidade de alguém construir uma casa, plantar ou fazer qualquer tipo de melhoria em terreno da união o direito de posse. Mas a primeira coisa que me vem a cabeça é de como faríamos esse cadastro de posse e essa ideia de desenhar o país por meio de polígonos é muito interessante e vou além. Deveria ter em cada polígono um CPF ou CNPJ cadastrado indicando posse. A entidade dos cartórios, que detém o garantimento do direito de propriedade é falha e extremamente inacessível, não sei já tiveste a oportunidade de ver uma escritura, mas tenho certeza que todo programador ao ver uma tem o mesmo pensamento que eu.

**Comment by thiagovasconcelosti** [comment]
Eu trabalho atualmente para a prefeitura do Natal-RN, mais especificamente para a secretaria de meio ambiente e urbanismo, e atualmente um dos serviços que a prefeitura está tentando transformar em algo acessível é algo como isso, com mais recursos, como áreas de preservação ambiental, áreas militares e outras divisões de zonas... No mesmo mapa é possível selecionar os serviços por marcação como: escolas, áreas em construção, Hospitais, centros de comércio e outros pontos selecionáveis e divididos... ao exemplo disso as prefeituras de [SP ](https://www.prefeitura.sp.gov.br/cidade/secretarias/pessoa_com_deficiencia/mapa_de_servicos/)e [Recife ](https://esigportal.recife.pe.gov.br/arcgis/apps/webappviewer/index.html?id=679e74b46c7b44caaad64abd5f751b4b)possuem algo semelhante. Basicamente são layers que são importadas usando ArcGIS web, essas layers são exportadas como json, o que facilita muito o trabalho, mas as informações ainda são limitadas dentro da secretaria no caso da minha cidade, e é algo não tão acessível em outras cidades.

**Comment by natanael127** [comment]
Dois links aqui para acalorar a discussão.
https://dados.gov.br/dataset/malha-geometrica-dos-municipios-brasileiros
https://mapshaper.org

Fiz um teste aqui com o estado de MG e funcionou legal esse segundo site aí exporta pra JSON
imagina agora uma api cujas entradas são as coordenadas geográficas e as saídas qual município vc está (sim, o google tem isso, mas a gente pode fazer uma)
